model/swagger.yml

```diff
@@ -1051,54 +545,120 @@ paths:
+
+        notes: the blob fields are blocked unless specifically requrested and only for the */csv and application/kvp+csv types.
+      required: false
+      schema:
+        type: array
+        items:
+          type: string
+    Identified:
+      name: identifier
+      in: path
+      description: |
+        syntax: lidvid or lid
+
+        behavior (lid): returns one or more items whose lid matches this lid exactly. If the endpoint ends with the identifier or /latest then a signle result is returned and it is the highest version. If the endpoint ends with /all then all versions of the lid are returned.
+
+        behavior (lidvid): returns one and only one item whose lidvid matches this lidvid exactly.
+
+        note: the current lid/lidvid resolution will match all the lids that start with lid. In other words, it acts like a glob of foobar*. It behaves this way from first character to the last
+
+        note: simple sorting of the lidvid is being done to select the latest from the end of the list. However, the versions 1.0, 2.0, and 13.0 will sort to 1.0, 13.0, and 2.0 so the end of the list may not be the latest.
+      required: true
+      schema:
+        type: string
+    Keyword:
+      name: keywords
+      in: query
+      description: |
+        syntax: keyword=keyword1,keyword2,...
+
+        behavior: free text search on title and description (if set q is ignored
+
+        notes: is this implemented?
+      required: false
+      schema:
+        type: array
+        items:
+          type: string
+    Limit:
+      name: limit
+      in: query
+      description: |
+        syntax: limit=10
+
+        behavior: maximum number of matching results returned, for pagination
+      required: false
+      schema:
+        type: integer
+        minimum: 1
+        default: 100
+    Query:
+      name: q
+      in: query
+      description: |
+        syntax: q="vid eq 13.0"
+
+        behavior: query uses eq,ne,gt,ge,lt,le,(,),not,and,or operators. Properties are named as in 'properties' attributes, literals are strings between quotes, like "animal", or numbers. Detailed query specification is available at https://bit.ly/3h3D54T
+
+        note: ignored when keyword is present
+      required: false
+      schema:
+        type: string
+    Sort:
+      name: sort
+      in: query
+      description: |
+        syntax: sort=asc(field0),desc(field1),...
+
+        behavior: is this implemented?
```

Comment:
I don't believe it is implemented. Let write that here.

Reply:
Oh, I need to remove that because I did implement it. Fixed.